### PR TITLE
Improve error messages around pipeline option

### LINKED ISSUE #8276: "The pipeline operator requires a proposal set error" during compile
My build crashes with: ` The pipeline operator requires a proposal set. You must pass the 'pipelineProposal' option to @babel/preset-stage-0` error. I modified webpack.config.js by adding that flag to presets but it still not work.

```presets: [
              ['@babel/stage-0', {
                pipelineProposal: true,
                decoratorsLegacy: true,
              }],
              '@babel/react',
              '@babel/preset-env',
            ],
```

Anyone know how to fix it?

### PR DESCRIPTION
**Pull request by mAAdhaTTah**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | Fixes #8276, #8277
| Patch: Bug Fix?          | Yes
| Major: Breaking Change?  | No
| Minor: New Feature?      | No
| Tests Added + Pass?      | Yes
| Documentation PR Link    | N/A
| Any Dependency Changes?  | N/A
| License                  | MIT

 Improve the error messages so we (hopefully) get fewer issues about this.

Thoughts about linking to the docs in the error message? Good or bad idea?

### CODE REVIEW

File: packages/babel-preset-stage-0/src/index.js [hunk 1]
```diff
@@ -38,8 +39,9 @@ export default declare((api, opts = {}) => {
   if (typeof pipelineProposal !== "string") {
     throw new Error(
       "The pipeline operator requires a proposal set." +
-        " You must pass the 'pipelineProposal' option to" +
-        " @babel/preset-stage-0",
+        " You must pass 'pipelineProposal' option to" +
+        " @babel/preset-stage-0 with one of: " +
```
Comment (nicolo-ribaudo) [inline packages/babel-preset-stage-0/src/index.js]: `with` -> `whose value must be`

**Comment by mAAdhaTTah** [comment]
Updated!

File: packages/babel-plugin-syntax-pipeline-operator/src/index.js [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
 export default declare((api, { proposal }) => {
   api.assertVersion(7);
 
   if (typeof proposal !== "string" || !proposals.includes(proposal)) {
-    throw new Error("'proposal' must be one of: " + proposals.join(", "));
+    throw new Error(
+      "The pipeline operator plugin requires a 'proposal' option." +
+        "'proposal' must be one of: " +
+        proposals.join(", ") +
+        ". More details: https://babeljs.io/docs/en/next/babel-plugin-syntax-pipeline-operator.html",
```
Comment (hzoo) [inline packages/babel-plugin-syntax-pipeline-operator/src/index.js]: I think you want to link to https://babeljs.io/docs/en/next/babel-plugin-proposal-pipeline-operator?

Comment (mAAdhaTTah) [reply packages/babel-plugin-syntax-pipeline-operator/src/index.js]: Whoops, fixed.

**Comment by mengli404** [comment]
@mAAdhaTTah after this branch merged to master, my error message still like this:
```
Build failed: [BABEL] /Users/mengliding/Documents/front_end_repo/network-map/.webpackrc.js: 'proposal' must be one of: minimal (While processing: "/Users/mengliding/Documents/front_end_repo/network-map/node_modules/_@babel_plugin-proposal-pipeline-operator@7.0.0-beta.52@@babel/plugin-proposal-pipeline-operator/lib/index.js$inherits")
Error: [BABEL] /Users/mengliding/Documents/front_end_repo/network-map/.webpackrc.js: 'proposal' must be one of: minimal (While processing: "/Users/mengliding/Documents/front_end_repo/network-map/node_modules/_@babel_plugin-proposal-pipeline-operator@7.0.0-beta.52@@babel/plugin-proposal-pipeline-operator/lib/index.js$inherits")
    at _default (/Users/mengliding/Documents/front_end_repo/network-map/node_modules/_@babel_plugin-syntax-pipeline-operator@7.0.0-beta.52@@babel/plugin-syntax-pipeline-operator/lib/index.js:26:11)
    at /Users/mengliding/Documents/front_end_repo/network-map/node_modules/_@babel_helper-plugin-utils@7.0.0-beta.52@@babel/helper-plugin-utils/lib/index.js:19:12
    at loadDescriptor (/Users/mengliding/Documents/front_end_repo/network-map/node_modules/_@babel_core@7.0.0-beta.52@@babel/core/lib/config/full.js:157:14)
    at cachedFunction (/Users/mengliding/Documents/front_end_repo/network-map/node_modules/_@babel_core@7.0.0-beta.52@@babel/core/lib/config/caching.js:32:19)
    at loadPluginDescriptor (/Users/mengliding/Documents/front_end_repo/network-map/node_modules/_@babel_core@7.0.0-beta.52@@babel/core/lib/config/full.js:192:28)
    at cache.invalidate.data (/Users/mengliding/Documents/front_end_repo/network-map/node_modules/_@babel_core@7.0.0-beta.52@@babel/core/lib/config/full.js:216:47)
    at CacheConfigurator.invalidate (/Users/mengliding/Documents/front_end_repo/network-map/node_modules/_@babel_core@7.0.0-beta.52@@babel/core/lib/config/caching.js:149:17)
    at instantiatePlugin (/Users/mengliding/Documents/front_end_repo/network-map/node_modules/_@babel_core@7.0.0-beta.52@@babel/core/lib/config/full.js:216:28)
    at cachedFunction (/Users/mengliding/Documents/front_end_repo/network-map/node_modules/_@babel_core@7.0.0-beta.52@@babel/core/lib/config/caching.js:32:19)
    at loadPluginDescriptor (/Users/mengliding/Documents/front_end_repo/network-map/node_modules/_@babel_core@7.0.0-beta.52@@babel/core/lib/config/full.js:192:10)
    at config.plugins.map.descriptor (/Users/mengliding/Documents/front_end_repo/network-map/node_modules/_@babel_core@7.0.0-beta.52@@babel/core/lib/config/full.js:68:16)
    at Array.map (<anonymous>)
    at recurseDescriptors (/Users/mengliding/Documents/front_end_repo/network-map/node_modules/_@babel_core@7.0.0-beta.52@@babel/core/lib/config/full.js:67:38)
    at recurseDescriptors (/Users/mengliding/Documents/front_end_repo/network-map/node_modules/_@babel_core@7.0.0-beta.52@@babel/core/lib/config/full.js:86:27)
    at recurseDescriptors (/Users/mengliding/Documents/front_end_repo/network-map/node_modules/_@babel_core@7.0.0-beta.52@@babel/core/lib/config/full.js:86:27)
    at loadFullConfig (/Users/mengliding/Documents/front_end_repo/network-map/node_modules/_@babel_core@7.0.0-beta.52@@babel/core/lib/config/full.js:100:6)
```

**Comment by nicolo-ribaudo** [comment]
Can you try to redownload and rebuild the repo?

**Pull request merged**